boards/hamilton/Makefile.include

```diff
@@ -0,0 +1,13 @@
+# define the cpu used by Hamilton
+export CPU = samd21
+export CPU_MODEL = samr21e18a
+
+# debugger config
+export JLINK_DEVICE := atsamr21e18a
+export TERMPROG := JLinkRTTClient
```

Comment:
RIOT already uses pyterm as a client for RTT. Please change to

```diff
diff --git a/boards/hamilton/Makefile.include b/boards/hamilton/Makefile.include
index f301c2731..432c79321 100644
--- a/boards/hamilton/Makefile.include
+++ b/boards/hamilton/Makefile.include
@@ -4,7 +4,8 @@ export CPU_MODEL = samr21e18a
 
 # debugger config
 export JLINK_DEVICE := atsamr21e18a
-export TERMPROG := JLinkRTTClient
+TERMPROG = $(RIOTTOOLS)/jlink/jlink.sh
+TERMFLAGS = term_rtt
 export OBJDUMPFLAGS += --disassemble --source --disassembler-options=force-thumb
 export OFLAGS := -O binary --gap-fill 0xff
 ```

Optionally, you might add the usage of `JLinkRTTClient` as a non-default setting to use it separately.